packages/react/src/components/List/List.tsx

```diff
@@ -1108,6 +1144,10 @@ export class List<T = any> extends React.Component<IListProps<T>, IListState<T>>
     const scrollHeight = getScrollHeight(this._scrollElement);
     const scrollTop = getScrollYPosition(this._scrollElement);
 
+    const doUpdate =
+      Math.abs((this._visibleRect?.bottom ?? 0) - (this._materializedRect?.bottom ?? 0)) <
+      this._estimatedPageHeight / 5;
```

Comment:
Is this still needed? It's commented out in line `1161` below.

Reply:
Removed